api/python/tests/integration/test_packages.py

```diff
@@ -319,7 +319,20 @@ def test_load_into_quilt(self):
             expected_params={
                 'Body': ANY,
                 'Bucket': 'my_test_bucket',
-                'Key': 'Quilt/package/foo',
+                'Key': 'Quilt/package/foo1',
+                'Metadata': {'helium': '{}'}
```

Comment:
I guess because of https://github.com/quiltdata/quilt/blob/ee5ccbdfcb026fe265ba3fc102465c92e99441aa/api/python/tests/integration/test_packages.py#L40 we cover the case with absolutely no metadata?